imbi/endpoints/integrations/google.py

```diff
@@ -96,10 +96,11 @@ async def sync_user(self, openid_token) -> imbi.user.User:
             raise errors.Forbidden('Invalid Google domain sign in')
         user_id = id_info['sub']
         user_email = id_info['email']
-        user_name = id_info['name']
-        imbi_user = imbi.user.User(self.application, username=user_email,
+        display_name = id_info['name']
+        username = user_email.split('@')[0]
```

Comment:
NIT -- `local_part, _, domain_part = user_email.partition('@')` is a little teeny bit cleaner since it handles icky non-email things with multiple `@` signs.  Just a not for other places where you might use similar code for more complex things.